src/ngx_http_lua_ssl_module.c

```diff
@@ -226,12 +226,25 @@ ngx_http_lua_ffi_update_ticket_encryption_key(SSL_CTX *ctx,
      * key. */
     if (keys->nelts > 0) {
         pkey = keys->elts;
-        if (ngx_memcmp(pkey->name, key, 16) == 0 &&
-            ngx_memcmp(pkey->aes_key, key + 16, 16) == 0 &&
-            ngx_memcmp(pkey->hmac_key, key + 32, 16) == 0)
-        {
-            dd("duplicate ticket key");
-            return NGX_OK;
+        if (ngx_strlen(key) == 48) {
+            dd("key size is 48");
+            if (ngx_memcmp(pkey->name, key, 16) == 0 &&
+                ngx_memcmp(pkey->aes_key, key + 16, 16) == 0 &&
+                ngx_memcmp(pkey->hmac_key, key + 32, 16) == 0)
+            {
+                    dd("duplicate ticket key");
+                    return NGX_OK;
+            }
+
+        } else if (ngx_strlen(key) == 80) {
+            dd("key size is 80");
+            if (ngx_memcmp(pkey->name, key, 16) == 0 &&
+                ngx_memcmp(pkey->aes_key, key + 16, 32) == 0 &&
```

Comment:
@fbomlisboa Good catch. That was actually not my code but some of my former colleagues'. Anyway, I've just committed a fix to the master branch with some more minor coding style issues found by the [ngx-releng](https://github.com/openresty/openresty-devel-utils/blob/master/ngx-releng) static analyzer tool.